docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -136,4 +136,34 @@ Below topologies from ONNX open model zoo are fully supported on OpenVINO Execut
 | --- | --- | --- | --- | --- |
 | tiny_yolov2 | Yes | Yes | Yes | Yes* |
 
-*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
\ No newline at end of file
+*FPGA only runs in HETERO mode wherein the layers that are not supported on FPGA fall back to OpenVINO CPU.
+
+## CSharp API
+
+To use csharp api for openvino execution provider create a custom nuget package.
+
```

Comment:
The instructions document how to run msbuild to create a couple of packages. Not clear what the user should do after that or what the expected output of the msbuild commands is. Are there two nuget packages produced that need to be manually copied somewhere and/or added as a reference to a C# project?